### ISSUE TITLE
Remmina Plugins doesn't work

### ISSUE THREAD

**Issue by khurshid-alam** [hugosenari/Kupfer-Plugins#4]
I was trying to use your remmina plugin, but it doesn't any show any host. It can only execute remmina app itself.

How does it suppose to work?

**Comment by hugosenari** [comment]
Thank you for fill this issue. 👍 

You can have these problems:
* You can't install plugin properly
    * please confirm if it is displayed and enabled in Kupfer Preferences->Plugins
* Plugin can't find your saved hosts
    * verify if directory defined at plugin's preference contains your data (*.remmina)
* Hosts aren't at Top Level index
    * In preferences->catalog check Remmina Hosts
    * Other option: in kupfer search for Remmina use "Search Content" instead of "Launch"
* Plugin can't interpret your remmina specific version
    * Save any simple hosts (with fake data) and send me to test here
* Code problem
    * When you activate plugin at preference they show you some error
* Code problem 2
    * Some problems only appears when kupfer try to index plugin sources, in this case run kupfer from console (maybe with --debug) to know what errors you have

Maybe this is good to include in a FAQ 😄

**Comment by khurshid-alam** [comment]
>Plugin can't find your saved hosts
verify if directory defined at plugin's preference contains your data (*.remmina)

It's that. Remmina configuration is saved under `/home/$USER/.remmina`. May be you want to make that default.

**Comment by hugosenari** [comment]
According to remmina [source code](https://github.com/FreeRDP/Remmina/blob/d8a48d31c912637987f130d3273d65839fed14bb/remmina/src/remmina_file_manager.c#L50):
* Legacy ~/.remmina, or
* ~/.local/share/remmina, or
* /usr/local/share/remmina

Since they define ~/.remmina as 'legacy', I wont make it default. 
Maybe search for both. What you think?

**Comment by khurshid-alam** [comment]
>Maybe search for both. What you think?

Yes. Search for both location would be nice. 

offtopic: I think this should go on upstream. Can you open a PR at https://github.com/KarolBedkowski/kupfer-adds (which is the new upstream) ? 

Only adding **remmina.py** in kupfer/plugins (https://github.com/KarolBedkowski/kupfer-adds/tree/master/kupfer/plugin) should be enough.  Thanks.

**Comment by hugosenari** [comment]
To be honest I would be happy if they accept my plugin at theirs main code.

But I don't think this is the correct answer, we need a marketplace for plugin.

Anyway I tried make install process frictionless with pip:
```bash
pip install kupfer_plugin_remmina
```
You don't need sudo since they only copy file to ~/.local/share/kupfer/plugin

Hoping that other developers could do the same, I created a [cookiecutter](https://github.com/hugosenari/cookiecutter-kupfer-plugin-package.git) to help them.

Maybe a plugin where user can search and install plugins from kupfer could be good idea :smile:
